nodemon.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Yes, I research that. Here's the actual implementation they reference in the issue: https://github.com/sgress454/sails-hook-autoreload

> Can't I just use forever or nodemon or [insert daemon here] to do this?
> **Yes, yes, you absolutely can**, and if stability during development is your #1 concern, you absolutely should. Nothing beats a cold reboot to get that oh-so-pristine app state. The advantages of this hook are (1) it's faster, (2) it won't run your bootstrap every time (see #1), and (3) it won't drop your sessions / socket connections if you're using in-memory adapters during development.

I just went with easier solution for now and it works just fine. However, when app grows large we definitely should look into speeding it up.